styles.css

```diff
@@ -5,40 +5,53 @@ html {
 }
 
 body {
+  display: flex;
+  background-color: #E7F1F1;
+}
+
+.random {
   height: 100vh;
-  width: 100vw;
+  width: 69vw;
   font-family: 'Poppins', sans-serif;
   display: flex;
   flex-direction: column;
   align-items: center;
   justify-content: space-evenly;
-  background-color: #E7F1F1;
+  border-right: solid 5px black;
 }
 
 h1 {
-  font-size: 7rem;
-  font-weight: 900;
+  height: 10%;
+  width: 100%;
+  font-size: 7em;
+  font-weight: 800;
```

Comment:
I would like to do some research on how to make font size more dynamic with the size of the screen.

Reply:
Great idea!